src/Components/Artwork/Details/Details.tsx

```diff
@@ -32,13 +33,9 @@ export interface DetailsProps {
   renderSaveButton?: (artworkId: string) => React.ReactNode
 }
 
-interface SaleInfoLineProps extends DetailsProps {
-  showActivePartnerOffer: boolean
-}
-
-interface SaleMessageProps extends DetailsProps {
-  showActivePartnerOffer: boolean
-}
+const LINE_HEIGHT = 22
+const NUM_OF_LINES = 5
+const CONTAINER_HEIGHT = LINE_HEIGHT * NUM_OF_LINES
```

Comment:
The current implementation requires logic to make sure we have consistent number of lines (e.g. if there is no badge, we will pad with an empty line). Since we agreed on fixed 5 lines, I feel it's easier to set a fixed container height and just add lines to it, without maintaining the # of lines logic which gets a bit tricky with client-side loaded content.